state_transfer/transport/stunnel/stunnel.go

```diff
@@ -4,19 +4,24 @@ import (
 	"bytes"
 	"fmt"
 
+	"github.com/konveyor/crane-lib/state_transfer/endpoint"
+	"github.com/konveyor/crane-lib/state_transfer/meta"
 	"github.com/konveyor/crane-lib/state_transfer/transport"
 	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	corev1 "k8s.io/api/core/v1"
 )
 
 const (
-	stunnelPort         = int32(1443)
-	stunnelImage        = "quay.io/konveyor/rsync-transfer:latest"
-	stunnelConfigPrefix = "crane2-stunnel-config-"
-	stunnelSecretPrefix = "crane2-stunnel-secret-"
+	stunnelImage  = "quay.io/konveyor/rsync-transfer:latest"
```

Comment:
Add a todo: this needs to be configurable before 1.6.0, to be compatible with MTC?